### ISSUE TITLE
MongoKitten decodes an error where the status is `ok` and logs unclearly

### ISSUE THREAD

**Issue by Joannis** [OpenKitten/MongoKitten#187]
```[ ERROR ] Couldn't find type String at path "["errmsg"]" (ErrorMiddleware.swift:26)
[ DEBUG ] Conform `BSONValueNotFound` to `Debuggable` for better debug info. (ErrorMiddleware.swift:26)```

Unknown scenario. The error's origin should be defined more clearly

**Comment by Joannis** [comment]
Fixed in 5.1.3

**Comment by Joannis** [comment]
Still causing issues
